### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by oimou**
### Description
Add unit tests for vuex store and Instanbul coverage.

### How to check
1. `yarn` to install dependencies.
2. `yarn test:unit`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,17 +31,24 @@
     "@vue/cli-plugin-e2e-nightwatch": "3.0.0-beta.15",
     "@vue/cli-plugin-eslint": "3.0.0-beta.15",
     "@vue/cli-plugin-unit-jest": "3.0.0-beta.15",
```
Comment (laSinteZ) [inline package.json]: Shouldn't we remove Jest if we are using Mocha? Or maybe we can use Jest for everything? Could you explain this, it seems strange for me to have 2 test frameworks.

Comment (laSinteZ) [reply package.json]: Also, please add `coverage` and `.nyc_output` to gitignore

File: tests/unit/store/Account.spec.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+        nodeIp: MOCK_NODE_IP,
+        accountInfo: {},
+        rawAssetTransactions: {},
+        assets: [],
+        connectionError: null
+      }
+
+      mutations[types.RESET](state)
+
+      expect(state).to.deep.equal(expectedState)
```
Comment (laSinteZ) [inline tests/unit/store/Account.spec.js]: `.to.deep` => `.to.be.deep` in several places (not a mistake of course, just small comment)

File: tests/unit/store/Account.spec.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+    it('GET_ACCOUNT_ASSET_TRANSACTIONS_SUCCESS should set transactions to state', () => {
+      const state = { rawAssetTransactions: {} }
+      const assetId = randomAssetId()
+      const transactions = MOCK_TRANSACTIONS
+
+      mutations[types.GET_ACCOUNT_ASSET_TRANSACTIONS_SUCCESS](state, { assetId, transactions })
+
+      expect(state.rawAssetTransactions)
+        .to.have.property(assetId)
+        .that.deep.equal(transactions)
```
Comment (laSinteZ) [inline tests/unit/store/Account.spec.js]: `.that.is.deep` :)

File: package.json (hunk 1 above)
Comment (oimou) [reply package.json]: It is totally my mistake since Jest is no longer used. I will remove Jest dependencies and config files :)

**Comment by laSinteZ** [comment]
BTW, let's develop in "develop" branch. I've forgot about it :(

**Pull request merged**